README.md

```diff
@@ -2,6 +2,10 @@
 
 Integrate react-intl with gettext tools. This package provides a CLI tool to help converting between react-intl JSON files and gettext PO files.
 
+## Installation
+
+```npm install react-intl-gettext --save-dev```
```

Comment:
- [ ] change markdown syntax to enable shell syntax highlighting
- [ ] add instructions for [yarn](https://yarnpkg.com/en/)